simtools/schemas/model_parameters/corsika_particle_kinetic_energy_cutoff.schema.yml

```diff
@@ -0,0 +1,52 @@
+%YAML 1.2
+---
+title: Schema for corsika_particle_kinetic_energy_cutoff model parameter
+version: 0.1.0
+meta_schema: simpipe-schema
+meta_schema_url: https://raw.githubusercontent.com/gammasim/simtools/main/simtools/schemas/model_parameter_and_data_schema.metaschema.yml
+meta_schema_version: 0.1.0
+name: corsika_particle_kinetic_energy_cutoff
+description: |-
+  Kinetic energy cutoffs for different particle types.
+data:
+  - name: hadrons
+    type: float64
+    description: Kinetic energy cutoffs for hadrons.
+    unit: GeV
+    default: 0.3
+    allowed_range:
+      min: 0.05
+    required: true
+  - name: muons
+    type: float64
+    description: Kinetic energy cutoffs for muons.
+    unit: GeV
+    default: 0.1
+    allowed_range:
+      min: 0.05
+    required: true
+  - name: electrons
+    type: float64
+    description: Kinetic energy cutoffs for electrons.
+    unit: GeV
+    default: 0.020
+    allowed_range:
+      min: 0.05
```

Comment:
min should be lower than default, did you mean 0.005?

Reply:
Well spotted! Corrected.


![Screenshot 2024-07-11 at 13 51 03](https://github.com/gammasim/simtools/assets/14001688/11817c96-5f68-4cef-8232-991af30a2388)

Reply:
Ok good! Interesting it is that low for electrons and photons.

Reply:
Yes - but note that this is for CORSIKA and some detectors are sensitive to such low energies. For Cherenkov, the emission threshold is 20 MeV at sea level.